targets/TARGET_TI/TARGET_MSP432/TODO

```diff
@@ -0,0 +1,13 @@
+- Ticker32 Klasse für msp432
+- Konfigurierbarer Backchannel UART
+- HFXT mit 24/12/6/3 MHz
+- ggf MCLK mit DCO (overclocking) und SMCLK mit HFXT ...
+- ADC internal channel
+- ADC klären ob es konflikt beim lesen/wandeln gibt...
+
+- RTC API
+- FLASH API
+- CRC API
+- SLEEP API
+
```

Comment:
I'm guessing this file was not intended to be here.

Reply:
The changes were done because of name clashes. The file msp432p401r.h is the official register definition file as distributed by TI. Changes to this file are therefore not the first choice, because this file will be updated on a regular basis. In line 1353 you find the CS macro for the Clock System (CS) subsystem. Without the changes in the mbed OS code you will get compile errors. The same is true for DIR_H (line 605).

Reply:
The TODO file was there by mistake - just deleted it ...